src/animation/KeyframeTrack.js

```diff
@@ -64,11 +64,22 @@ Object.assign( KeyframeTrack, {
 
 			};
 
-			var interpolation = track.getInterpolation();
+			// Assumptions for custom interpolations:
+			// 1) The constructor is a part of the THREE module at the time of serialization
+			// 2) The constructor will be a part of the THREE module at the time of deserialization
+			if ( track.hasCustomInterpolation() ) {
```

Comment:
Another significant assumption here is that `constructor.name` will be stable with minification. Unfortunately I don't think that is true, and `THREE.MyClass.constructor.name !== 'MyClass'` is possible, although perhaps it doesn't affect the Editor?

Reply:
The Editor doesn't use the minified version of THREE js so it wouldn't affect the editor. However, I think we'd want this to work with all versions anyway.

I could add a property to store the class name of all `Interpolants` similar to how all instances of `Object3D` have a `type` property to store their names.